### DISCUSSION TITLE
Image ref.

### DISCUSSION THREAD

**Discussion by officegeek** [thuliteio/doks#814] [Q&A] [answered]
Hi
I have a problem wit reference a image.

the md file and the image are in the same folder.

**sql.md
test.jpg**

I use ![](test.jpg) 

But when building the site I get an _Image not found_ error

What are I'm doing wrong?

Best
Tue

**Comment by h-enk** [comment, marked as answer]
Make sure your image is in a [page bundle](https://gohugo.io/content-management/page-bundles/) (see also the Doks docs: [Add a local image](https://getdoks.org/docs/how-to/images-and-media/images/#add-a-local-image)), like so:

```bash
..
├── database/
│   ├── sql/
│   │   ├── index.md
│   │   └── DTU_Workbench.jpg
│   └── _index.md
└── _index.md
```

**Reply by officegeek** [reply]
Hi
Thanks for your answer :-)

Creating the sql folder, renaming to index.md and moving the image into the sql folder did the trick - Thanks a lot :-)

Best
Tue
